src/commands/test/test-story.spec.js

```diff
@@ -0,0 +1,102 @@
+const fs = require('fs-extra');
+const { getImageDiffer } = require('../../diffing');
+const testStory = require('./test-story');
+
+jest.mock('fs-extra');
+jest.mock('../../diffing');
+
+describe('testStory', () => {
+  const target = { captureScreenshotForStory: jest.fn() };
+  const tolerance = 'mock tolerance';
+  const configuration = 'mock configuration';
+  const configurationName = 'Configuration';
+  const kind = 'Kind';
+  const story = 'Story';
+  const filename = `${configurationName}_${kind}_${story}.png`;
+
+  const executeWithOptions = options =>
+    testStory(
+      target,
+      options,
+      tolerance,
+      configuration,
+      configurationName,
+      kind,
+      story
+    );
+
+  beforeEach(target.captureScreenshotForStory.mockReset);
+
+  describe('reference image is missing', () => {
+    beforeEach(() => {
+      fs.pathExists.mockReturnValueOnce(Promise.resolve(false));
+    });
+
+    it('throws an error if requireReference option is true', async () => {
+      const options = { requireReference: true };
+
+      return expect(executeWithOptions(options)).rejects.toThrow(
```

Comment:
Nice! 🤠 
Consider adding `expect.assertions(1)` to make sure the assertions was called during a test.

Reply:
More info in the links below:

- Jest / Expect / expect.assertions(number)
https://jestjs.io/docs/en/expect.html#expectassertionsnumber

- An Async Example
https://jestjs.io/docs/en/tutorial-async